nodepool/files/nodepool.yml

```diff
@@ -154,3 +164,26 @@ diskimages:
       DIB_GRUB_TIMEOUT: '0'
       DIB_DISTRIBUTION_MIRROR: 'http://mirror.rackspace.com/ubuntu'
       DIB_DEBIAN_COMPONENTS: 'main,universe'
+  - name: rpco-14.2-xenial-artifacts
```

Comment:
This image is intended to replace https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/rpc_openstack.yml#L343 with something built and managed by nodepool, rather than relying on something that was a once-off and not repeatable.